src/App.js

```diff
@@ -233,7 +223,7 @@ class App extends React.Component {
                 <ListSidebar
                   mainContent={<SubmissionList />}
                   sideComponents={[
-                    <SubFilterSidebar/>,
+                    <SubFilterSidebar key={"subFilterSidebar"} />,
```

Comment:
```suggestion
                    <SubFilterSidebar key={"SubFilterSidebar"} />,
```
Viết hoa ký tự đầu cho giống mấy cái key còn lại e